node/src/service.rs

```diff
@@ -124,67 +150,98 @@ pub fn new_partial(
 ///
 /// This is the actual implementation that is abstract over the executor and the runtime api.
 #[sc_tracing::logging::prefix_logs_with("Parachain")]
-async fn start_node_impl<RB>(
-	parachain_config: Configuration,
-	collator_key: CollatorPair,
-	polkadot_config: Configuration,
-	id: ParaId,
-	validator: bool,
-	rpc_ext_builder: RB,
+async fn start_node_impl<RuntimeApi, Executor, RB, BIQ, BIC>(
+    parachain_config: Configuration,
+    polkadot_config: Configuration,
+    id: ParaId,
+    rpc_ext_builder: RB,
+    build_import_queue: BIQ,
+    build_consensus: BIC,
 ) -> sc_service::error::Result<(TaskManager, Arc<TFullClient<Block, RuntimeApi, Executor>>)>
-where
-	RB: Fn(
-			Arc<TFullClient<Block, RuntimeApi, Executor>>,
-		) -> jsonrpc_core::IoHandler<sc_rpc::Metadata>
-		+ Send
-		+ 'static,
+    where
+        RuntimeApi: ConstructRuntimeApi<Block, TFullClient<Block, RuntimeApi, Executor>>
+        + Send
```

Comment:
suggest indent more similar to here https://github.com/substrate-developer-hub/substrate-parachain-template/blob/master/node/src/service.rs#L156

Reply:
added to https://github.com/DataHighway-DHX/node/issues/231